### PR TITLE
Update v12 pre-release files (1/N) 

### PR DESCRIPTION
**Pull request by ewafula**
#### What scientific question is your analysis addressing?
Generate analysis pre-release files for v12

#### What was your approach?
updated analysis pre-release modules and generated analysis pre-release files using  `scripts/generated-analysis-files.sh` bash script 

#### What GitHub issue does your pull request address?
NA

#### Which areas should receive a particularly close look?
Review update module code  and results

#### Is there anything that you want to discuss further?
None

#### Is the analysis in a mature enough form that the resulting figure(s) and/or table(s) are ready for review?
YES

#### What types of results are included (e.g., table, figure)?
results for the v12 pre-release modules 

#### What is your summary of the results?
results for the v12 pre-release modules 

#### Reproducibility Checklist

- [x] The dependencies required to run the code in this pull request have been added to the project Dockerfile.
- [x] This analysis has been added to continuous integration.

#### Documentation Checklist

- [x] This analysis module has a `README` and it is up to date.
- [x] This analysis is recorded in the table in `analyses/README.md` and the entry is up to date.
- [x] The analytical code is documented and contains comments.

### CODE REVIEW

File: analyses/fusion-summary/01-fusion-summary.Rmd [hunk 1]
```diff
@@ -99,13 +103,6 @@ starfusionDF <- read_tsv(file.path(dataDir, "fusion-starfusion.tsv.gz"))
 # read in DGD data 
 dgd_fusion <- readr::read_tsv(file.path(dataDir, "fusion-dgd.tsv.gz"))
```
Comment (jharenza) [inline analyses/fusion-summary/01-fusion-summary.Rmd:103]: For lines 94-104 here, I think we need a new approach. What this is doing is that it is taking any sample which had a fusion call in any gene and adding it to the fusion dataframes. BUT, what I think we want is _any sample assayed for fusions_ in these dataframes. Any which had RNA-Seq or DGD fusion panel. Can you update this? It is needed downstream for the HGG subtyping (possibly LGG).

Comment (jharenza) [reply analyses/fusion-summary/01-fusion-summary.Rmd:103]: Note: I updated this

**Comment by ewafula** [comment]
Updated analysis pre-release files for reharmonized RNAseq data regenerated and uploaded to s3 bucket v12 folder, including the latest  efo-mondo-map.tsv in [PR #350](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/pull/350)

**Review by jharenza — APPROVED** [review]

**Comment by jharenza** [comment]
All checks have passed, so this is ready to merge upon confirming data download is up to date and working properly cc @ewafula

**Comment by jharenza** [comment]
🚀

**Pull request merged**